certora/specs/SocialRecoveryModule.spec

```diff
@@ -63,8 +27,8 @@ function safeIsModuleEnabled(address module) returns bool {
 }
 
 // A summary function to check if an address is an owner in the Safe contract.
-function safeIsOwner(address calledContractAddress, address owner) returns bool {
-    assert calledContractAddress == safeContract; // This checks that the contract is Safe Smart Account.
+function safeCallerCheckIsOwner(address calledContractAddress, address owner) returns bool {
```

Comment:
Oh, I see, this is function is used as a summary.

I would suggest using a prefix (like we did for the `require` setups) for this as well:

```suggestion
function sumarizeSafeIsOwner(address callee, address owner) returns bool {
```

Reply:
Updated 👍🏾